rir/src/interpreter/interp.cpp

```diff
@@ -296,6 +298,8 @@ static RIR_INLINE SEXP createLegacyArgsList(const CallContext& call,
     }
 }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-variable"
```

Comment:
You can also just completely remove those asserts. Those are really from the beginning of time, when I wasn't exactly sure how setjmp/longjmp work.

Reply:
If they would fail, the C compiler would have been very drunk...